.gitignore

```diff
@@ -15,3 +15,5 @@ results
 
 npm-debug.log
 node_modules
+
+rsvg-convert.exe
```

Comment:
You should have your own personal `.gitignore` for that. https://help.github.com/articles/ignoring-files/#create-a-global-gitignore